openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml

```
@@ -83,22 +77,26 @@ properties:
                   The sender address. It must be a **verified** address. # todo
                   add a link to help docs on how to verify an email address
                 maxLength: 254
+                example: example@example.com
               subject:
                 type: string
                 description: The message subject. The template placeholders are allowed.
                 maxLength: 998
+                example: Sujet de démonstration
               text:
                 type: string
                 description: |
                   The message's text body.
                   Leave empty to use content from "html".
                   The template placeholders are allowed.
+                example: Texte de démonstration
               html:
                 type: string
                 description: |
                   The message's html body.
                   Leave empty to use content from "text".
                   The template placeholders are allowed.
+                example: <p>Texte de démonstration</p>
```

```suggestion
                example: "<p>Texte de démonstration</p>"
```

I wonder if tests did not failed because of tag

---

Please elaborate? Here is the preview https://preview.redoc.ly/rebilly-users/fix-broadcast-message-example/tag/Broadcast-Messages#operation/PostBroadcastMessage

---

https://github.com/Rebilly/api-definitions/blob/e01f829eaaf7d5be2e7ee975a42c687f405f22c0/openapi/components/schemas/EmailMessages/EmailMessage.yaml#L81

---

nvm, I just guessed